CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## New Features
 
+- PR #787: homogeneity, completeness and v-measure metrics ml-prim
```

Comment:
I believe you've missed merging latest changes from branch-0.9 to this PR and then migrating this changelog statement to under the 0.9 release section.